integrationtests/src/main/java/io/kroxylicious/proxy/testkafkacluster/KafkaClusterConfig.java

```diff
@@ -5,13 +5,9 @@
  */
 package io.kroxylicious.proxy.testkafkacluster;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Properties;
-import java.util.TreeMap;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.*;
```

Comment:
Nit: better not to use `*` imports and explicitly import what is needed. 

It should just be an IDE setting.

Reply:
done